metafix/src/test/java/org/metafacture/metafix/MetafixRecordTest.java

```diff
@@ -1462,11 +1462,9 @@ public void renameArrayOfHashes() {
     }
 
     @Test
-    @MetafixToDo("Do we actually need/want implicit $append? WDCD?")
     public void copyIntoArrayOfHashesImplicitAppend() {
```

Comment:
Hm, the `$append` is explicit (original meaning), but the array is implicit (new meaning). Maybe change these test names to `...(Implicit|Explicit)Array(Append|Prepend)...`?

Reply:
Right, thanks, I didn't actually remember what the original goal of that test was. Changed the names as you suggested.